R/relevel.R

```diff
@@ -26,7 +26,10 @@ flatten_operators <- function(pd_nested) {
 #' @keywords internal
 flatten_operators_one <- function(pd_nested) {
   pd_token_left <- c(special_token, math_token, "'$'")
-  pd_token_right <- c(special_token, "LEFT_ASSIGN", "'+'", "'-'")
+  pd_token_right <- c(
+    special_token, "LEFT_ASSIGN", if (parser_version_get() > 1) "EQ_ASSIGN",
```

Comment:
Not sure. A few changes in #441 did not have any known effects on the styling, it's just about making the AST look the same. But probably it would break something because there was some reason why it was not included in the list I guess.